js/docEnums.js

```diff
@@ -276,7 +276,7 @@
 
 /**
  * @typedef {string} Enums.FilterBuilderFieldFilterOperations
- * @enum {'='|'<>'|'<'|'<='|'>'|'>='|'contains'|'endswith'|'isblank'|'isnotblank'|'notcontains'|'startswith'}
+ * @enum {'='|'<>'|'<'|'<='|'>'|'>='|'between'|'contains'|'endswith'|'isblank'|'isnotblank'|'notcontains'|'startswith'}
```

Comment:
To avoid a BC in the SMD we should add new value to the end of the list (after `'startswith'`).